modules/data-table/main/index.coffee

```diff
@@ -786,10 +786,18 @@ urlFeed = (url, options) ->
 
   {
     url: url # for debugging
-    headers: maybeCached (r) -> hx.json(url, { type: 'headers', extra: options.extra }, r)
-    totalCount: maybeCached (r) -> hx.json(url, { type: 'totalCount', extra: options.extra }, (res) -> r(res.count))
-    rows: (range, cb) -> hx.json(url, { type: 'rows', range: range, extra: options.extra }, cb)
-    rowsForIds: (ids, lookupRow, cb) -> hx.json(url, { type: 'rowsForIds', ids: ids, extra: options.extra }, cb)
+    headers: maybeCached (cb) ->
+      hx.json url, { type: 'headers', extra: options.extra }, (err, res) ->
+        cb(res)
```

Comment:
An alternative would be to write your own url feed that has error handling but it makes sense to add it here - I didn't really think about it because it wasn't doing it before but I'll add it in